packages/core-ui/styles/modal.scss

```diff
@@ -1,6 +1,77 @@
 /* @license Copyright 2023 @paritytech/polkadot-dashboard-ui authors & contributors
 SPDX-License-Identifier: Apache-2.0 */
 
+.modal-footer {
+  display: flex;
+  align-items: center;
+  width: 100%;
+
+  h3 {
+    color: var(--text-color-secondary);
+    opacity: 0.5;
+    position: relative;
+    top: 1.25rem;
+
+    &.active {
+      color: var(--network-color-primary);
+      opacity: 1;
+    }
+  }
+}
+
+.modal-blur {
+  background: var(--overlay-modal-color);
+  position: fixed;
```

Comment:
position, width, height, backdrop-filter all can be a mixin and applied to this and `modal-wrapper`.